test/v2/lib/recovery-nocks.js

```diff
@@ -2484,3 +2484,29 @@ module.exports.nockXlmRecovery = function() {
     }
   });
 };
+
+module.exports.nockBtcSegwitRecovery = function() {
+  // Nock all the external api calls that gather info about the wallet
+  // We have lots of empty addresses, because the code queries for possible addresses in the wallet one by one
+  const emptyAddrs = ['2N42muVaEhvcyMRr7pmFPnrmprdmWCUvhy7', '2N2b2yNryWVbMjvXFq7RbaQ2xbGhmAuBQM7', '2NBs5i2APw3XSvfch7rHirYC6AxehYizCU9', '2NEFHeSYnHVt4t2KqwKz1AZqhpcx2yGoe38', '2N4iR1AweHV8wmc7VPBb3tRnweQs1fSW3dB', '2N1ir7htudeFEWGhyfXGL7LNKzoFrDS62bQ', '2NBpZak1Tz1cpLhg6ZapeTSHkhq91GwMYFo', '2N93AW6R6eLan8rfB715oCse9P6pexfK3Tn', '2NEZiLrBnTSrwNuVuKCXcAi9AL6YSr1FYqY'];
+  emptyAddrs.forEach(function(addr) {
+    nock('https://testnet-api.smartbit.fakeurl/v1')
+    .get('/blockchain/address/' + addr)
+    .reply(200, JSON.parse('{"success":true, "address":{"address":"' + addr + '", "total":{"received":"0", "received_int":0, "spent":"0", "spent_int":0, "balance":"0", "balance_int":0, "input_count":0, "output_count":0, "transaction_count":0}, "confirmed":{"received":"0", "received_int":0, "spent":"0", "spent_int":0, "balance":"0", "balance_int":0, "input_count":0, "output_count":0, "transaction_count":0}, "unconfirmed":{"received":"0", "received_int":0, "spent":"0", "spent_int":0, "balance":"0", "balance_int":0, "input_count":0, "output_count":0, "transaction_count":0}, "multisig":{"confirmed":{"balance":"0", "balance_int":0},"unconfirmed":{"balance":"0","balance_int":0}}}}'));
```

Comment:
A bit strange to JSON.parse() here, but it will work. You can also just give a JS object to nock as the reply value.

Reply:
It is strange, but it was far easier than replacing every double-quote with single-quotes, then removing the quotes around every key (all Lint requirements)

Reply:
Fair enough - but you can do this relatively easily by pasting the whole JSON.parse blob into a node REPL, and pasting it back in here, then formatting it (that will take care of the double quotes and quoted keys for you). I'll accept as is, but please do this in the future (code is written once, and read many times, so we should optimize for readability where possible).